tools/remove_grouping_from_report_results.rb

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+require 'trollop'
+
+opts = Trollop.options do
+  banner "Remove extras[:grouping] from MiqReportResult#report column.\n\nUsage: ruby #{$PROGRAM_NAME} [options]\n\nOptions:\n\t"
+  opt :dry_run,    "For testing, rollback any changes when the script exits.", :short => :none, :default => false
+  opt :batch_size, "Limit memory usage by process this number of report results at a time.",    :default => 50
```

Comment:
Does 50 batch size make sense?  Considering the size of these, I wanted extra queries instead of possibly bloating the memory of the process.

Reply:
I think so.  The deserialization is probably going to hit us harder than the queries time themselves.  I don't see this as being an issue with `N+1`'s.